### ISSUE TITLE
Progressive Ajax Loading : callback after each page or all pages loaded

### ISSUE THREAD

**Issue by etriponezf** [olifolkerd/tabulator#3389]
**Is your feature request related to a problem? Please describe.**
I am trying to progressively load data in a table using ajaxprogressiveload="load", and then want to reapply filters to the table when all pages are loaded. My server is sending data by page only for performance reasons, in Tabulator the data is not paginated. Currently I have found no way of knowing when all pages are loaded, the dataLoaded callback being called after the first page only. 
My related issue on StackOverflow: https://stackoverflow.com/questions/69178427/tabulator-progressive-ajax-loading-applying-a-filter-after-all-pages-are-rec

**Describe the solution you'd like**
I'd like to have a callback after each page is loaded and/or when all pages are loaded.

**Describe alternatives you've considered**
I tried the following:
- force filter in dataLoaded callback => doesn't work. The callback is called only after the first page
- force filter in pageLoaded callback => doesn't work, callback is not called
- force filter in rowAdded callback => works but as it's executed for every row, takes a while
In the meantime for the specific filter issue, I modified the tabulator.js code to get this to work, by adding this.table.refreshFilter(); after addRows() in Page.prototype._parseRemoteData.

However I also need to do other things when all pages are loaded, so I force-called the dataLoaded callback at the end of addRow**s** (RowManager.prototype.addRows). I understand that my solutions work in my specific case, but might not work if you allow users to manually add rows. It would be nice to have a specific callback somewhere after pages are loaded.
